mlflow/models/evaluation/base.py

```diff
@@ -456,11 +485,120 @@ def _start_run_or_reuse_active_run(run_id):
     else:
         if run_id and active_run.info.run_id != run_id:
             raise ValueError(
-                "An active run exists, you cannot specify another run_id when " "evaluating."
+                "An active run exists, you cannot specify another run_id when evaluating."
             )
         yield active_run.info.run_id
 
 
+def _normalize_evaluators_and_evaluator_config_args(
+        evaluators,
+        evaluator_config,
+):
+    from mlflow.models.evaluation.evaluator_registry import _model_evaluation_registry
+
+    def check_nesting_config_dict(_evaluator_name_list, _evaluator_name_to_conf_map):
+        return isinstance(_evaluator_name_to_conf_map, dict) and \
+               all(k in _evaluator_name_list and isinstance(v, dict)
+                   for k, v in _evaluator_name_to_conf_map.items())
+
+    if evaluators is None:
+        evaluator_name_list = list(_model_evaluation_registry._registry.keys())
+        if len(evaluator_name_list) > 1:
+            print(f'Multiple registered evaluators are found {evaluator_name_list} and '
```

Comment:
Do we want to use warning.warn() instead of stdout print so that it's easier for users to hide this message or send it to logs for automated frameworks that interface with this?

Reply:
@WeichenXu123 Can we use `_logger.warn()` / `_logger.info()` throughout this PR, instead of `print()`? We shouldn't use print in MLflow.